### ISSUE TITLE
Production build blocked by CORS!

### ISSUE THREAD

**Issue by Maarondesigns** [AnalyticalGraphicsInc/cesium#8286]
Our site is suddenly being blocked by CORS. Changing to a different build has no effect. This completely breaks our app (assuming the approximate terrain file). Can this please be fixed asap? This is breaking a production site for my company and if we can't rely on this framework we will have to move to a different product. I have a presentation of the app I've been building to the owner at the end of this week!! Please, thank you....
![cesiumscors](https://user-images.githubusercontent.com/40376601/66843701-2ab31880-ef3b-11e9-9495-f8e2fe3711e7.jpg)

**Comment by hpinkos** [comment]
Hello @Maarondesigns, we recently moved the releases from cesiumjs.org to cesium.com.

If you change your URL to `https://cesium.com/downloads/cesiumjs/releases/1.62/...` it should work, but please let me know if you are still experiencing problems as we just rolled this change out this morning.  Thanks!

**Comment by Maarondesigns** [comment]
Thank you for the quick response @hpinkos! It is working again however the skybox images are still being blocked, though this isn't a breaking change for me at the moment. Thanks!

**Comment by hpinkos** [comment]
Thanks for the update @Maarondesigns 
Are the skybox images giving a CORS error as well?

**Comment by Maarondesigns** [comment]
Yep

**Comment by Chingili** [comment]
Yes, the skybox is still getting blocked 
![image](https://user-images.githubusercontent.com/47533430/66847279-60063900-ef30-11e9-8e28-61ea20f4ea71.png)

**Comment by hpinkos** [comment]
Thanks @Chingili, @Maarondesigns, we are working to fix this as we speak and I'll give you an update once I have one.

**Comment by Chingili** [comment]
I also noticed 

https://cesium.com/downloads/cesiumjs/releases/1.62/Build/Cesium/Assets/IAU2006_XYS/IAU2006_XYS_16.json

https://cesium.com/downloads/cesiumjs/releases/1.62/Build/Cesium/Assets/approximateTerrainHeights.json

are also blocked. I think that is what causes polygons that are ->  clampToGround: true not to render.

**Comment by hpinkos** [comment]
@Chingili @Maarondesigns I believe we've fixed the CORS issue.  The skybox loaded correctly in my test application.  Could you confirm you are no longer seeing the errors?

**Comment by Chingili** [comment]
Thank you! Yes, my polygons are back. I am not sure what this

https://cesium.com/downloads/cesiumjs/releases/1.62/Build/Cesium/Assets/IAU2006_XYS/IAU2006_XYS_16.json

is used for, however it is still getting blocked.

**Comment by Maarondesigns** [comment]
I am no longer seeing any CORS errors, thank you!

**Comment by Chingili** [comment]
https://cesium.com/downloads/cesiumjs/releases/1.62/Build/Cesium/Assets/approximateTerrainHeights.json 

is getting blocked again. 

![image](https://user-images.githubusercontent.com/47533430/66970832-0abf4a00-f04c-11e9-90d5-e9388d3c33c7.png)

**Comment by hpinkos** [comment]
@Chingili can you try clearing your cache and see if you still see the error?  Thanks

**Comment by OmarShehata** [comment]
@hpinkos documentation on https://sandcastle.cesium.com/ doesn't seem to be working because of a CORS issue? Just tried it in an incognito window. Double click on any class/method and you get in the console:

```
Access to XMLHttpRequest at 'https://cesium.com/docs/cesiumjs-ref-doc/types.txt' from origin 'https://sandcastle.cesium.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

**Comment by hpinkos** [comment]
I'm seeing this now too.  I"m not sure what changed but I'll take a look

**Comment by thw0rted** [comment]
I won't bother filing a new issue, I guess, but be aware that the domain move also caused CORS failures for the "Video" Sandcastle demo.  Behavior is the same as #8213 -- maybe we need a new issue to handle Video loading failures more gracefully? (Or, if there are existing error handlers, hook them properly in the Video Sandcastle example.)

**Comment by OmarShehata** [comment]
Closing since this seems to be resolved. The video Sandcastle is working at the time of writing: https://sandcastle.cesium.com/index.html?src=Video.html
